src/components/SearchBar.js

```diff
@@ -0,0 +1,59 @@
+import styled from "styled-components/macro";
+import { useState } from "react";
+import { useHistory } from "react-router-dom";
+
+function SearchBar({ searchTerm, onChange }) {
+  const [isToggled, setIsToggled] = useState(false);
+  const history = useHistory();
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    setIsToggled(!isToggled);
+    history.push("/search");
+  }
+
+  return (
+    <Form onSubmit={handleSubmit}>
+      <label>
+        <ScreenReaderInfo>Search bar</ScreenReaderInfo>
+        <Input
+          type="text"
+          name="input"
+          placeholder="Suche nach PLZ oder Stadt"
+          isToggled={isToggled}
```

Comment:
What does `isToggled` do?

Reply:
Nothing 🙈 It's a holdover from my first version, when I was working with a button + icon. I forgot to delete the code after deciding to go just with an input field.